test/acceptance/features/bindKnativeService.feature

```diff
@@ -39,7 +39,10 @@ Feature: Bind knative service to a service
                 - name: DB_PASSWORD
                   value: "{{ .knav.status.dbCredentials.password }}"
       """
-    Then deployment must contain intermediate secret "binding-request-knative"
+    Then deployment must contain intermediate secret
+      | intermediate_secret_name |
+      | binding-request-knative  |
```

Comment:
why this change here?

Reply:
I have created a reusable step implementation for `Then deployment must contain intermediate secret` hence i changed here too. I am aware of the PR being related to etcd. but because of this [https://github.com/redhat-developer/service-binding-operator/pull/644#issuecomment-682049475](url) i thought to put it here.

Reply:
I would keep the old step where only one secret is mentioned and introduce if really needed additional step, e.g.

```
deployment must contain secrets X, Y, Z
```

btw, what is the meaning of "intermediate secreat" and why a user should care?

Reply:
added this because of https://github.com/redhat-developer/service-binding-operator/pull/567#discussion_r472015323

Reply:
not following.

Reply:
well, step [](url)(https://github.com/redhat-developer/service-binding-operator/blob/b15ec71a3aa146227a7e40e7e798a260eafb3449/test/acceptance/features/bindAppToMultipleServices.feature#L51)
and [](url)(https://github.com/redhat-developer/service-binding-operator/blob/b15ec71a3aa146227a7e40e7e798a260eafb3449/test/acceptance/features/bindKnativeService.feature#L42) 
can make a similar implementation.what if we need validate more than 2 secrets for binding. we don't need to add another function. hence while implementing [https://github.com/shvasude/service-binding-operator/blob/04673a8e8eebbd6e0a763d2867c18b33ed687331/test/acceptance/features/bindAppToService.feature#L219](url)
added this bdd feature which can be reusable across number of secrets.  
[](url)(https://github.com/shvasude/service-binding-operator/blob/04673a8e8eebbd6e0a763d2867c18b33ed687331/test/acceptance/features/bindAppToService.feature#L242)